tools/dist_train.sh

```diff
@@ -16,5 +16,4 @@ python -m torch.distributed.launch \
     --master_port=$PORT \
     $(dirname "$0")/train.py \
     $CONFIG \
-    --seed 0 \
```

Comment:
No need to remove?

Reply:
It seems the current argparse in train.py does not contain the seed variable.
https://github.com/open-mmlab/mmdetection3d/blob/dev-1.x/tools/train.py#L15